electrumx/server/session.py

```diff
@@ -1555,7 +1606,8 @@ def set_request_handlers(self, ptuple):
 
         if ptuple >= (1, 4, 2):
             handlers['blockchain.scripthash.unsubscribe'] = self.scripthash_unsubscribe
-
+        if ptuple >= (1, 4, 4):
+            handlers['blockchain.transaction.broadcast_package'] = self.package_broadcast
```

Comment:
Let's either not expose the new method at all, or maybe expose it unconditionally by moving it ~6 lines higher into the big dict.

I don't want to gate it behind an arbitrary temporary version number. Once https://github.com/spesmilo/electrum-protocol/pull/2 is done, we can gate it behind that version number along with the other changes.

Until it is documented in the spec, clients should not rely on this method existing at any version.